gamification/src/main/java/juja/microservices/gamification/user/UserRepositoryCustom.java

```diff
@@ -29,10 +29,9 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-package juja.microservices.gamification.model.repository;
+package juja.microservices.gamification.user;
```

Comment:
@sLisnychyi Have a look [here](https://github.com/vkuchyn/github-issue-stat/blob/master/src/main/java/juja/github/stat/mongo/TimeReportMongoDao.java)
I am not a big fan of repositories because they make us use interface as part of implementation and hide lot of stuff behind though make design not flexible. Even if so, we need to separate spring data interface from our business interface and Custom is not a good name. Better to name domain interface UserRepository. And SD - MongoUserRepository. Like so. But better to use only MongoTemplate and implement logic manually.